site/src/sphinx/setup.rst

```diff
@@ -3,8 +3,192 @@
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-grpc</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-jetty</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-kafka</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-logback</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-retrofit2</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-rxjava</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-saml</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-thrift</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-tomcat</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-zipkin</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+        <dependency>
+          <groupId>com.linecorp.armeria</groupId>
+          <artifactId>armeria-zookeeper</artifactId>
+          <version>\ |release|\ </version>
+        </dependency>
+
+        <!-- Logging -->
+        <dependency>
+          <groupId>ch.qos.logback</groupId>
+          <artifactId>logback-classic</artifactId>
+          <version>\ |ch.qos.logback:logback-classic:version|\ </version>
+          <scope>runtime</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.slf4j</groupId>
+          <artifactId>log4j-over-slf4j</artifactId>
+          <version>\ |org.slf4j:log4j-over-slf4j:version|\ </version>
+          <scope>runtime</scope>
+        </dependency>
+      </dependencies>
+    </project>
+
+Using Maven BOM for simpler dependency management
+-------------------------------------------------
+
+You can import ``com.linecorp.armeria:armeria-bom`` into your build rather than specifying Armeria versions in
+more than once place. See `this article <https://www.baeldung.com/spring-maven-bom>`_ to  learn more about what
```

Comment:
nit: `to  learn` -> `to learn` (removing space)

Reply:
Thanks! Fixed